packages/create-neutrino-project/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "create-neutrino-project",
+  "version": "1.0.0",
+  "description": "Scaffold new Neutrino projects with a CLI helper",
+  "bin": {
+    "create-neutrino-project": "./bin/create-neutrino-project.js"
+  },
+  "scripts": {
+    "start": "create-neutrino-project init"
+  },
+  "files": [
+    "bin",
+    "commands"
+  ],
+  "repository": "https://github.com/mozilla-neutrino/create-neutrino-project",
```

Comment:
The repository format here will need to be updated to be a format similar to what the rest of the monorepo packages define.